tools/pipelines/dependency-injection/_package.json

```diff
@@ -3,6 +3,6 @@
     "version": "0.0.1",
     "description": "Install dependencies on pipeline VM for DI",
     "dependencies": {
-        "@ff-internal/aria-logger": "^0.0.7"
+        "@ff-internal/aria-logger": "^0.0.8"
```

Comment:
can this just be ^0.0.0 so we take it automatically? i would even consider just *

Reply:
I did `>0.0.0` and tested that this excludes pre-release versions, which is what I want.  `^` might have worked too 🤷‍♂️